File: AOJ/ITP1_re/itp_1_4_d.cpp
#include <bits/stdc++.h>

int main() {
  long n, a, min, max, sum;
  scanf("%ld", &n);
  for (int i = 1; i <= n; i++) {
    scanf("%ld", &a);
    if (i == 1) {
      min = a;
      max = a;
      sum = a;
      continue;
    }
    if (a < min) {
      min = a;
    }
    if (max < a) {
      max = a;
    }
    sum += a;
  }
  printf("%ld %ld %ld\n", min, max, sum);
}

File: AOJ/ITP1_re/itp_1_4_d.go
package main

import "fmt"

func main() {
	var n, a, min, max, sum int
	fmt.Scan(&n)
	for i := 1; i <= n; i++ {
		fmt.Scan(&a)
		if i == 1 {
			min = a
			max = a
			sum = a
			continue
		}
		if a < min {
			min = a
		}
		if max < a {
			max = a
		}
		sum += a
	}
	fmt.Println(min, max, sum)
}
